adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureRDD.scala

```diff
@@ -95,10 +102,117 @@ object FeatureRDD {
 
     FeatureRDD(rdd, sd)
   }
+
+  /**
+   * @param feature Feature to convert to GTF format.
+   * @return Returns this feature as a GTF line.
+   */
+  private[features] def toGtf(feature: Feature): String = {
+    def escape(entry: (Any, Any)): String = {
+      entry._1 + " \"" + entry._2 + "\""
+    }
+
+    val seqname = feature.getContigName
+    val source = Option(feature.getSource).getOrElse(".")
+    val featureType = Option(feature.getFeatureType).getOrElse(".")
+    val start = feature.getStart + 1 // GTF/GFF ranges are 1-based
+    val end = feature.getEnd // GTF/GFF ranges are closed
+    val score = Option(feature.getScore).getOrElse(".")
+    val strand = Features.asString(feature.getStrand)
+    val frame = Option(feature.getFrame).getOrElse(".")
+    val attributes = Features.gatherAttributes(feature).map(escape).mkString("; ")
+    List(seqname, source, featureType, start, end, score, strand, frame, attributes).mkString("\t")
+  }
+
+  /**
+   * @param feature Feature to write in IntervalList format.
+   * @return Feature as a one line interval list string.
+   */
+  private[rdd] def toInterval(feature: Feature): String = {
+    val sequenceName = feature.getContigName
+    val start = feature.getStart + 1 // IntervalList ranges are 1-based
+    val end = feature.getEnd // IntervalList ranges are closed
+    val strand = Features.asString(feature.getStrand, emptyUnknown = false)
+    val attributes = Features.gatherAttributes(feature)
```

Comment:
I'm not 100% here—I'm a bit unfamiliar with the IntervalList format, even after this PR—but the implementation here passes round trip (with the exception of some ordering bits) with the interval list file in our test resources directory. For me to understand the difference a bit better, could you perhaps elucidate how this code would change the output? E.g., how would the resultant interval list line look different?